recipes/unitig-counter/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "unitig-counter" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnlees/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: e91e726bb9d11e25868bac990b55d9f178ae195a826992dd0be6e4508011f4a7
+
+build:
+  number: 0
+  skip: True #  [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python <3
```

Comment:
Please put python in the host section.

Reply:
Not needed, now boost is being pulled from conda-forge